compiler/record-minmax/include/RecordMinMax.h

```diff
@@ -58,6 +61,8 @@ class RecordMinMax
   luci_interpreter::Interpreter *getInterpreter() const { return _interpreters[0].get(); }
   MinMaxObserver *getObserver() const { return _observers[0].get(); }
 
+  std::vector<std::vector<std::vector<char>>> importH5Data(const std::string &input_data_path);
```

Comment:
the `vector` is not friendly to read.
is there any particular reason to use so?

Reply:
Actually no, just convenience.
And then what is better to use in this case?

Reply:
> And then what is better to use in this case?

Not using three vectors.

Why not use unique_ptr to memory for inner most vector<char> ? 
or refer recent PR uses with `using` alias; https://github.com/Samsung/ONE/pull/10234/files#diff-e4360f1df0c1003cc097406de284d98870516a540ebdd4b19181e1a4d25852a3